code/game/objects/structures/inflatable.dm

```diff
@@ -163,6 +166,9 @@
 		return !opacity
 	return !density
 
+/obj/structure/inflatable/door/CanAtmosPass(turf/T)
```

Comment:
I would have sworn this was already defined somewhere, but I may be thinking of CanPass().